### PR TITLE
Feature/fmfr 197 spreadsheet frontend i

### PR DESCRIPTION
**Pull request by ramchurnk**

### CODE REVIEW

File: app/views/facilities_management/procurements/_about_the_work.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <h2 class="govuk-heading-m govuk-!-margin-bottom-1">About the work</h2>
-<%= link_to 'Bulk upload', new_facilities_management_procurement_spreadsheet_import_path(procurement_id: @procurement.id) %>
+<%= link_to 'Click here to download an Excel template', facilities_management_procurement_path(procurement_id: @procurement.id, 'spreadsheet': true) %>, which you can complete and then upload, or click on the links below to complete your information.
```
Comment (doctorpod) [inline app/views/facilities_management/procurements/_about_the_work.html.erb:2]: Use I18n

File: app/views/facilities_management/procurements/_spreadsheet.html.erb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  </ol>
+  <div class="govuk-body govuk-!-font-weight-bold govuk-!-margin-top-2 govuk-!-width-two-thirds">
+    <%= t('.footer_information') %>
+  </div>
+  <div>
+    <%= form_for @procurement, url: facilities_management_procurement_path(@procurement.id), method: :patch do |f| %>
+      <%= f.submit 'Continue to upload your file', name: 'bulk_upload_spreadsheet', class: 'govuk-button govuk-button--primary' %>
+      <%= f.submit 'Save for later', name: 'bulk_upload_spreadsheet', class: 'govuk-button govuk-button--secondary govuk-!-margin-left-4' %><br/>
+      <%= f.submit( 'Cancel and return to procurement summary',
+                   tabindex:-1,
```
Comment (doctorpod) [inline app/views/facilities_management/procurements/_spreadsheet.html.erb:41]: Use I18n

File: app/views/facilities_management/procurements/spreadsheet_imports/show.html.erb [hunk 3]
```diff
@@ -1,16 +1,8 @@
...
-<% end %>
-
+<%= render partial: 'facilities_management/procurements/spreadsheet_imports/upload_status' %>
 
-<%= link_to 'Go to procurement', facilities_management_procurement_path(@spreadsheet_import.procurement), class: 'govuk-link' %>
\ No newline at end of file
+<% if @spreadsheet_import.importing? %>
+  <%= link_to 'Return to procurements dashboard', facilities_management_procurements_path, class: 'govuk-link govuk-!-margin-top-4' %>
+  <% else %>
+  <%= link_to 'Go to procurement', facilities_management_procurement_path(@spreadsheet_import.procurement), class: 'govuk-link govuk-!-margin-top-4' %>
```
Comment (doctorpod) [inline app/views/facilities_management/procurements/spreadsheet_imports/show.html.erb:7]: Use I18n

File: app/views/facilities_management/procurements/_about_the_work.html.erb (hunk 1 above)
Comment (ramchurnk) [reply app/views/facilities_management/procurements/_about_the_work.html.erb]: done

File: app/views/facilities_management/procurements/_spreadsheet.html.erb (hunk 2 above)
Comment (ramchurnk) [reply app/views/facilities_management/procurements/_spreadsheet.html.erb:41]: done

File: app/views/facilities_management/procurements/spreadsheet_imports/show.html.erb (hunk 3 above)
Comment (ramchurnk) [reply app/views/facilities_management/procurements/spreadsheet_imports/show.html.erb]: done

**Pull request merged**